styles/stylesheet.css

```diff
@@ -0,0 +1,446 @@
+  background-size: cover;
+  color: #FFFFFF;
+  height: 26em;
+}
+
+.header-text {
+  font-size: 1.2rem;
+  font-style: italic;
+  color: green;
+}
+
+.checkout-box {
+  margin: 0;
+  display: block;
+}
+
+.logo-img {
+  width: 40%;
+}
+
+.cart-img {
+  width: 2em;
+  margin-right: .5rem;
+}
+
+.checkout-button-top {
+    background-color: green;
+    color: #FFFFFF;
+    height: 3em;
+    width: 100%;
+    padding: 1em 2em;
+    border-radius: 0.2em;
+}
+
+/* NAVIGATION */
+
+nav {
+  background-color: #d1d1d1;
+  height: 3.9rem;
+}
+
+nav ul {
+  padding: 1.125em 0;
+  width: 90%;
+  margin: auto;
+}
+
+li {
+  padding: 1.125em 0;
+  display: inline;
+  list-style-type: none;
+  text-decoration: none;
+}
+
+li a {
+  text-decoration: none;
+  color: #1c4282;
+  font-size: 1.1rem;
+  font-weight: bold;
+  padding: 1em;
+}
+
+/* MAIN SECTION */
+
+.format {
+display: flex;
+  margin: 0 1em;
+}
+
+
+.main-section {
+  margin-top: 3em;
+  width: 100%;
+}
+
+article {
+  width: 100%;
+}
+
+.main-image {
+  margin-left: 0;
+  margin-bottom: 1em;
+  width: 100%;
+}
+
+.main-body {
+  margin-top: 3em;
+ display: flex;
+ flex-direction: column;
+ justify-content: flex-start;
+ flex-wrap: wrap;
+}
+
+h2 {
+  color: #1c4282;
+  padding: 2rem 0;
+}
+
+aside {
+  margin-top: 6em;
+  padding-left: 3em;
+}
+
+
+.blog-preview-pic {
+  width: 100%;
+}
+
+.blog-snippet {
+  margin: 1rem 0 3em;
+ line-height: 1.5em;
+}
+
+.preview-header {
+  color: #1c4282;
+}
+
+.preview-link {
+  color: #a3c3f7;
+}
+
+p {
+  margin: .5rem 0 1rem;
+}
+
+/* DOG PROFILESf */
+
+.dog-profiles {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(2, auto);
+  grid-auto-flow: row;
+  grid-gap: 3rem;
+  font-size: 0.8rem;
+}
+
+.dog-profiles-2 {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: repeat(3, auto);
+  grid-auto-flow: row;
+  grid-gap: 5em;
+  font-size: 0.8rem;
+}
+
+.dog {
+  display: block;
+  width: 22em;
+  border: 0.1em solid #c7c8c9;
+}
+
+.dog:hover {
+  background-color: #ddeaff;
+}
+
+.chb {
+  margin: auto;
+  width: 100%;
+  background-color: green;
+  color: #FFFFFF;
+  padding: 1em 1em;
+  border-radius: 0.2em;
+  text-align: center;
+}
+
+.dog-pic {
+  width: 100%;
+}
+
+.dog-description {
+  padding: .5em;
+  margin-bottom: .5rem;
+
+}
+
+.cost-to-adopt {
+  font-weight:bold;
+}
+
+.profile-header {
+  color: #1c4282;
+  font-weight: bold;
+}
+
+.see-more {
+  margin: auto;
+  width: 20%;
+}
+
+.see-more-button {
+  background-color: #e3e4e5;
+  color: #1c4282;
+  font-weight: bold;
+  text-align: center;
+ max-width: 15rem;
+ padding: .5rem 0;
+ margin: 3rem auto;
+}
+
+/* FOOTER */
+
+.footer-section {
+  background: linear-gradient(#073B60, #000000);
+  text-align: center;
+  font-family: arial;
+  color: #FFFFFF;
+  margin-top: 3.8em;
+  padding: 2em;
+}
+
+.footer-logo {
+  max-width: 20%;
+}
+
+/*BLOG PAGE */
+
+#blog-post-container {
```

Comment:
it's generally not good practice to use id's for CSS. It's totally approprate to create a single class. We try to use ID's for javascript needs.